### DISCUSSION TITLE
ILI9342 display with weberbox pcb

### DISCUSSION THREAD

**Discussion by mtredeye** [nebhead/PiFire#166] [General]
Looking for guidance on which display to use for the Weberbox PCB, I ordered the 3.2 ILI9341 (using nebhead box) that was linked in the PiFire guide, but all I get is a grey screen then eventually it turns off...looking at the reviews on Amazon for that screen, one of the reviews said he could only get it to work on 3.3v...which was the design of the V2a pcb that is connected to 3.3V for the VCC on the display, however the weberbox PCB is connected to 5v for vcc on the display. I have been looking at different screens with a few saying if using 5v to use level shifters in their description.

**Comment by nebhead** [comment]
Have you enabled the display via the configuration wizard?  

Using these displays with either 5V or 3.3V should work.  I believe they are tolerant of both voltages and may be a bit brighter with 5V.  Your display is probably fine, but there is probably some other issue going on.

**Reply by mtredeye** [reply]
It was a bad screen, got a replacement from Amazon this morning and it came right on! That one review had me concerned on Amazon...but all good now!
